plugins/command-line/prism-command-line.js

```diff
@@ -35,41 +96,30 @@ Prism.hooks.add('complete', function (env) {
 	}
 
 	// Create the "rows" that will become the command-line prompts. -- cwells
-	var lines = new Array(1 + env.code.split('\n').length);
+	var promptLines = new Array(1 + env.code.split('\n').length);
 	var promptText = pre.getAttribute('data-prompt') || '';
 	if (promptText !== '') {
-		lines = lines.join('<span data-prompt="' + promptText + '"></span>');
+		promptLines = promptLines.join('<span data-prompt="' + promptText + '"></span>');
 	} else {
 		var user = pre.getAttribute('data-user') || 'user';
 		var host = pre.getAttribute('data-host') || 'localhost';
-		lines = lines.join('<span data-user="' + user + '" data-host="' + host + '"></span>');
+		promptLines = promptLines.join('<span data-user="' + user + '" data-host="' + host + '"></span>');
 	}
 
 	// Create the wrapper element. -- cwells
 	var prompt = document.createElement('span');
 	prompt.className = 'command-line-prompt';
-	prompt.innerHTML = lines;
+	prompt.innerHTML = promptLines;
 
-	// Mark the output lines so they can be styled differently (no prompt). -- cwells
-	var outputSections = pre.getAttribute('data-output') || '';
-	outputSections = outputSections.split(',');
-	for (var i = 0; i < outputSections.length; i++) {
-		var outputRange = outputSections[i].split('-');
-		var outputStart = parseInt(outputRange[0]);
-		var outputEnd = outputStart; // Default: end at the first line when it's not an actual range. -- cwells
-		if (outputRange.length === 2) {
-			outputEnd = parseInt(outputRange[1]);
-		}
-
-		if (!isNaN(outputStart) && !isNaN(outputEnd)) {
-			for (var j = outputStart; j <= outputEnd && j <= prompt.children.length; j++) {
-				var node = prompt.children[j - 1];
-				node.removeAttribute('data-user');
-				node.removeAttribute('data-host');
-				node.removeAttribute('data-prompt');
-			}
+	// Remove the prompt from the output lines. -- cwells
+	outputRanges.forEach(function (range) {
+		for (var i = range.start; i <= range.end; i++) {
```

Comment:
I chose `forEach()` here because it made the code easier to read for anyone coming behind me.  My initial code was actually a nested `for` loop with `i` and `j` variables.  I rewrote it using `forEach()` and it looked cleaner and easier to read.